.github/workflows/test.yml

```diff
@@ -17,29 +17,19 @@ jobs:
   test: # make sure the action works on a clean machine without building only on PRs
     if: ${{ github.event_name == 'pull_request' }}
     runs-on: ubuntu-latest
+    env:
+      tf_version: 0.12.27
     steps:
     - uses: actions/checkout@v1
+    - uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: ${{ env.tf_version }}
+    - run: terraform init
+      working-directory: __tests__
+    - run: terraform plan -out test-plan.tfplan
+      working-directory: __tests__
+    - run: cp -r __tests__/.terraform .
```

Comment:
I guess `terraform show` needs access to the .terraform directory. That's too bad. Does this have the potential of stomping on a .terraform directory in `.`?

Reply:
You're right. I tweaked the action so that it does the copying of the .terraform dir to the root. That still doesn't solve if there is a current .terraform dir at the root before this action runs. Honestly I don't know how we can get around this until terraform changes how the `terraform show` command works

Reply:
Got it!